srcpkgs/ugdb/template

```diff
@@ -1,14 +1,16 @@
 # Template file for 'ugdb'
 pkgname=ugdb
-version=0.1.9
+version=0.1.10
 revision=1
 build_style=cargo
+depends="gdb"
+checkdepends="gdb"
 short_desc="Alternative TUI for gdb"
 maintainer="Érico Nogueira <ericonr@disroot.org>"
 license="MIT"
 homepage="https://github.com/ftilde/ugdb"
 distfiles="${homepage}/archive/${version}.tar.gz"
-checksum=c96925da24fa684f6b22f1fee90a83d34c69be0466ae2fd7fa96cd5af7e903a1
+checksum=ccc41aed24f3ed8fd18decbcbb12ffa78542b5de37692a6dc927c08e4eb234bb
 
 case "$XBPS_TARGET_MACHINE" in
 	ppc*) broken="nix-0.7.0 fails to build";;
```

Comment:
This should truly be removable this time around.

Reply:
If I'm not overseeing something [here](https://github.com/nix-rust/nix), ppc is still not officially supported by nix though.

Reply:
Well, but it builds :p 

0.7.0 was the problematic release that didn't even build.

Reply:
Okay, removing it